### ISSUE TITLE
Fix inconsistent `urlpath` links to directories in tests and docs

### ISSUE THREAD

**Issue by kadykov** [intake/intake#764]
In the documentation and in test files you can find different ways to link to the subdirectory.
Like this:
```yaml
      urlpath: '{{ CATALOG_DIR }}/cache_data/states.csv'
```
and like that:
```yaml
      urlpath: '{{ CATALOG_DIR }}cache_data/states.csv'
```
What do you think if we keep only one way, for instance the first one with implicit `/`?
Probably, it will be safer and easier to maintain...

**Comment by martindurant** [comment]
Yes, I think the first of these feels more right.

**Comment by kadykov** [comment]
Fixed by #765
